modules/labs-react/common/README.md

```diff
@@ -64,3 +64,35 @@ export const HelloWorld = ({backgroundImagePath}) => {
   );
 };
 ```
+
+## useThemedRing
+
+This is a way to automatically add themed colors to your inputs. This is helpful when showing alerts to users. It supports `error`, `alert`, and `success` states. It will try and use the corresponding `main` colors from your CanvasThemePalette, however if they do not meet accessibility contrast the outer ring with use the `darkest` color. This hook will also show a `focusOutline` ring when the input is focused.
```

Comment:
```suggestion
This is a way to automatically add themed colors to your inputs. This is helpful when showing alerts to users. It supports `error`, `alert`, and `success` states. It will try and use the corresponding `main` colors from your `CanvasThemePalette`, however if they do not meet accessibility contrast the outer ring with use the `darkest` color. This hook will also show a `focusOutline` ring when the input is focused.
```